src/utils/logging.ts

```diff
@@ -35,7 +38,10 @@ function createChildLogger (name: string, root: Debugger): Logger {
     debug: extendLogger({ logger: root, name: 'debug', log: consoleLog, enabled: root.enabled }),
     child: (name: string) => {
       const child = root.extend(name);
-      child.enabled = root.enabled;
+      if (root.enabled) {
```

Comment:
The old approach of explicitly enabling logging on a single logger broke with the upgraded `debug` package